packages/strapi-helper-plugin/lib/src/components/LeftMenuList/List.js

```diff
@@ -5,7 +5,7 @@ import colors from '../../assets/styles/colors';
 const List = styled.ul`
   margin-bottom: 0;
   padding-left: 0;
-  max-height: 251px;
+  max-height: 180px;
```

Comment:
Are you sure that modifying the style of this components does not impact the `Settings` ?